### ISSUE TITLE
CAM generations

### ISSUE THREAD

**Issue by gozdedemirci** [ucuapps/WSMIS#5]
Hello again,

I try to follow directories to generate CAM. In your paper, you mentioned you used both GradCAM and GradCAM++, then decided to use GradCAM++. In your inference/cam_generation folder, there is generate_pascal.py file which I believe this file I should run to extract CAM. However, I see that you don't call either GradCAM and GradCAM++ in this file. Did you do some changes on GradCAM and used your own? Or should I go to another file?

Also, I am running the classification model (model_training/cam_generation/train_pascal.py) on a custom dataset by adjusting dataloaders. During training, I guess it is overfitting. My training loss is low whereas validation loss increases up to 33. My dataset is ~15.000 images. Did you also face that issue? Do you have any suggestions?

Lastly, am I referring correct directories? Thank you very much!

**Comment by Mugen-chen** [comment]
I have the same question. Do you find where the gradcam++ is used？
